src/api_tracker.py

```diff
@@ -0,0 +1,68 @@
+import os
+import requests
+from ckan.common import config
+import ckan.plugins.toolkit as tk
+import hashlib
+import logging
+import re
+
+
+log = logging.getLogger(__name__)
+
+def ga_api_tracker(resource_id):
+    try:
+        tid = config.get('googleanalytics.id', None)
+        endpoint = config.get('googleanalytics.endpoint', None)
+        client_id = hashlib.md5(tk.c.user).hexdigest()
+        payload = {
+            'v': 1, # Version.
```

Comment:
i would store the dataset `dataset.name`

Reply:
So I leave it as is atm, e.g. PCA_2013, EPD_202002, correct?
The names are unique (including dataset info - PCA, EPD)